KhangTPH-verify-the-2018-Honda-Civic-Values-Cars-for-Sale-Kelley-Blue-Book-page-should-be-displayed-after-click-Get-Reviews-button

truongkhang0905 opened the pull request:
Delete target folder
Create ReviewCarsPage in CarResearchPages folder
Create ResearchNewAndUsedCarsPageTest in CarResearchPagesTests folder, 
delete some file which is not relate to the project

Commit:
Delete target folder, Create ResearchNewAndUsedCarsPage, ReviewCarsPage in CarResearchPages folder, Create ResearchNewAndUsedCarsPageTest in CarResearchPagesTests folder, delete some file which is not relate to the project

doanhoangtri committed:
update

Commit:
Delete target file

namnguyen-itvn commented on demo/src/test/java/com/example/test/pages/KBBMOOCProject/CarResearchPages/ResearchNewAndUsedCarsPage.java:
```diff
@@ -0,0 +1,54 @@
+        //TODO Auto-generated constructor stub
+    }
+
+    WebElement selectYear = keyword.findWebElementByXpath("//select[@placeholder='Year']");
```
pls change.....

namnguyen-itvn replied:
....

namnguyen-itvn commented on demo/src/test/java/com/example/test/pages/KBBMOOCProject/CarResearchPages/ResearchNewAndUsedCarsPage.java:
```diff
@@ -0,0 +1,54 @@
+
+    WebElement selectYear = keyword.findWebElementByXpath("//select[@placeholder='Year']");
+    WebElement selectMake = keyword.findWebElementByXpath("//div[@class='make']//select[@placeholder='Make']");
+    WebElement selectModel = keyword.findWebElementByXpath("//div[@class='model']//select[@placeholder='Model']");
```
....

namnguyen-itvn replied:
update have been pushed

Commit:
- Delete target file
- Edit :
   + Datadriven file
   + Set up web element  outside method/function
   + Customkeyword file, no use thread.sleep inside

Commit:
Update code, delete target file,
